cmd/lotus/daemon.go

```diff
@@ -400,7 +400,7 @@ func ImportChain(ctx context.Context, r repo.Repo, fname string, snapshot bool)
 		defer resp.Body.Close() //nolint:errcheck
 
 		if resp.StatusCode != http.StatusOK {
-			return xerrors.Errorf("non-200 response: %d", resp.StatusCode)
+			return xerrors.Errorf("Import chain failed with non-200 response: %d", resp.StatusCode)
```

Comment:
```suggestion
			return xerrors.Errorf("fetching chain CAR failed: %d", resp.StatusCode)
```

Reply:
Keeping the reason (non-200 response) would help users diagnose the problem.